contracts/WitnetBridgeInterface.sol

```diff
@@ -94,12 +94,9 @@ contract WitnetBridgeInterface is VRF {
     payable
     payingEnough(msg.value, _tallyReward)
   returns(uint256 _id) {
-    _id = uint256(sha256(_dr));
-    if(requests[_id].dr.length != 0) {
-      requests[_id].tallyReward += _tallyReward;
-      requests[_id].inclusionReward += msg.value - _tallyReward;
-      return _id;
-    }
+    _id = requests.length;
+    DataRequest memory dr;
+    requests.push(dr);
```

Comment:
Is this kind of initialization needed at all? I suspect this could just work:
```solidity
_id = requests.length;
requests[_id].dr = _dr;
requests[_id].inclusionReward = msg.value - _tallyReward;
```

Reply:
I don't think so, accessing elements with index >= array.length results in an "Invalid Opcode" exception.